src/renderer/styles/notification-settings.less

```diff
@@ -3,112 +3,181 @@
 @color: rgba(0, 0, 0, .8);
 
 html {
-  margin: 0;
-  height: 100%;
+  margin     : 0;
+  height     : 100%;
   font-family: @font-family;
 }
 
 body {
-  margin: 0;
-  height: 100%;
+  margin     : 0;
+  height     : 100%;
   font-family: @font-family;
 }
 
 .content {
   border-radius: 2px;
-  width: 100%;
-  flex: 0 0 auto;
+  width        : 100%;
+  flex         : 0 0 auto;
 }
 
 .header {
-  display: flex;
-  align-items: center;
-  line-height: 1.3;
+  display        : flex;
+  align-items    : center;
+  line-height    : 1.3;
   justify-content: space-between;
-  border-bottom: 1px solid rgba(0, 0, 0, .1);
-  margin-bottom: 20px;
-  padding: 16px;
+  border-bottom  : 1px solid rgba(0, 0, 0, .1);
+  margin-bottom  : 20px;
+  padding        : 16px;
 }
 
-.header__title {
-  font-weight: 500;
-  font-style: normal;
-  margin: 0 0 0 4px;
-  font-size: 1.4rem;
-  min-height: 13px;
+.header-title {
+  font-weight  : 300;
+  font-style   : normal;
+  margin       : 0 0 0 4px;
+  font-size    : 1.4rem;
+  min-height   : 13px;
   text-overflow: ellipsis;
-  text-align: center;
-  white-space: nowrap;
-  overflow: hidden;
-  width: 100%;
-  color: @color;
+  text-align   : center;
+  white-space  : nowrap;
+  overflow     : hidden;
+  width        : 100%;
+  color        : @color;
```

Comment:
Is this spacing required?

Reply:
Oh, it looks like the formatter has included that :) I'll fix it.